README.md

```diff
@@ -1,2 +1,63 @@
 # mobilefirstportfolio
 This is a portfolio website that has prioritized mobile user  experience over Desktop user.
+## Built With
+
+- HTML and CSS.
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+-Coming soon
+
+
+## Getting Started
+
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [ ] Please review this part of your README file. Kindly add a descriptive title and description of your app. Write steps about how to use the app, the right technologies used, and remove unused sections. Try to make this more professional looking so it will catch more attention of possible recruiters in the future.